contrib_bots/lib/giphy/giphy.py

```diff
@@ -0,0 +1,140 @@
+# To use this plugin, you need to set up the Giphy API key for this bot in
+# ~/.giphy_config
+
+from __future__ import absolute_import
+from __future__ import print_function
+import six
+import requests
+import logging
+import sys
+import os
+import re
+if six.PY2:
+    from six.moves.configparser import SafeConfigParser
+else:
+    from configparser import SafeConfigParser
+
+GIPHY_TRANSLATE_API = 'http://api.giphy.com/v1/gifs/translate'
+
+if not os.path.exists(os.environ['HOME'] + '/.giphy_config'):
```

Comment:
If someone would import this file to inspect its classes and exceptions and whatnot they would have to have a `.giphy_config` ready even though they didn't have any intention of using it. I know that this file isn't imported anywhere but normally it is a good practice to do these kinds of checks in the `__main__` method if there is any or only where it's necessary. Just a side note.